README.md

```diff
@@ -0,0 +1,77 @@
+# Recent stolen bikes & search widget
```

Comment:
2 things:

1. Why switch the filename here? It would be nice to keep it the same so the history is more easily browseable
2. can you add a screenshot?

Reply:
Def agree good catch.  I changed the filename and added a screenshot